subsys/bootloader/fprotect/fprotect_bprot.c

```diff
@@ -22,17 +24,17 @@
  /* The number of CONFIG registers present in the chip. */
 #define BPROT_CONFIGS_NUM ceiling_fraction(BPROT_REGIONS_NUM, BITS_PER_LONG)
 #if defined(CONFIG_SB_BPROT_IN_DEBUG)
-#define ENABLE_BPROT_IN_DEBUG true
+#define ENABLE_IN_DEBUG true
 #else
-#define ENABLE_BPROT_IN_DEBUG false
+#define ENABLE_IN_DEBUG false
```

Comment:
Should this be renamed just `ENABLE` ?

Reply:
No I don't think so. Even if `ENABLE_IN_DEBUG` is false, it will still be enabled if no debugger is attached.

Reply:
Okei :)